dynamics/test.py

```diff
@@ -230,6 +229,21 @@ def dynamics_client(request) -> MockClient:  # pylint: disable=W0621
         yield request.param
 
 
+@pytest.fixture
+def environ():
```

Comment:
Hmm, this should probably be in `tests/conftest.py`, since otherwise it will be included in the pip wheel. I don't think it's necessary for other people to get this fixture for their testing.

Reply:
Ah! That's what I was missing (and the reason I had imported environ "manually") - chalk it up to not being familiar enough yet with pytest. 

Thanks again.

Reply:
Importing it there also works, but I meant you should move it to `tests/conftest.py`, since it's not _really_ part of the dynamics-client specific testing suite, and shouldn't be included with the pip wheel on release.

Reply:
Got it - I moved it instead of simply importing into conftest. This way it won't be exposed in the dynamics namespace.